### DISCUSSION TITLE
Cyberduck will not allow uploads

### DISCUSSION THREAD

**Discussion by Mek-magnet** [iterate-ch/cyberduck#15879] [Q&A]
I have repaired the softeware, restarted numerous times and finally uninstalled and reinstalled and after connecting to my sites I am not able to upload any files.

**Comment by tatze96** [comment]
- Which version of Cyberduck is in use?
- Which protocol is in use?
- Are there any error messages? If so, please attach a screenshot of it to this ticket.

**Sidenote**: Make sure to attach the screenshot through the GitHub web interface, as it wouldn't be visible to us otherwise.

**Comment by Mek-magnet** [comment]
I uninstalled my previous version after trying t repair it and installed the newest version. Even when I try to see the About Cyberduck to get you the exact revision number it does not work. I am trying to use the FTP version to connect to three separate sites. There are no error messages, I just get the spinning wait circle indefinitely. I am able to connect to each site, but when I try to upload I get a blank popup and no functions work.

Monika
Team MEK

**Reply by tatze96** [reply]
It sounds like you are running out of user handles. 

This is likely a duplicate for #15885. Please follow the workaround within the mentioned ticket and let us know if this improves the behavior.
